Code/GraphMol/Atom.cpp

```diff
@@ -517,6 +517,80 @@ void Atom::invertChirality() {
       break;
   }
 }
+
+namespace MDL {
```

Comment:
I think these should be in different files not necessarily in Atom.h, but I like the current api

C++

```
RDKit::MDL::setRLabel(atom, 1);
RDKit::Daylight::setSupplementalLabel(atom, "<foobar>");
```

python:

```
Chem.MDL.SetRLabel(atom, 1)
Chem.Daylight.SetSupplementalLabel(atom, "<foobar>")
```

I think having these explicit will make the more discoverable and not as hidden as they used to be.